pkg/bundle/controller.go

```diff
@@ -130,14 +67,21 @@ func AddBundleController(ctx context.Context, mgr manager.Manager, opts Options)
 
 		// Reconcile a Bundle on events against a ConfigMap that it
 		// owns. Only cache ConfigMap metadata.
-		WatchesMetadata(&corev1.ConfigMap{}, handler.EnqueueRequestForOwner(mgr.GetScheme(), mgr.GetRESTMapper(), &trustapi.Bundle{}, handler.OnlyControllerOwner())).
+		Watches(&corev1.ConfigMap{}, handler.EnqueueRequestForOwner(
+			mgr.GetScheme(),
+			mgr.GetRESTMapper(),
+			&trustapi.Bundle{},
+			handler.OnlyControllerOwner(),
+		), builder.OnlyMetadata).
+
+		////// Sources //////
 
 		// Reconcile trust.cert-manager.io Bundles
-		WatchesRawSource(&source.Informer{Informer: bundleInformer}, &handler.EnqueueRequestForObject{}).
+		Watches(&trustapi.Bundle{}, &handler.EnqueueRequestForObject{}).
 
 		// Watch all Namespaces. Cache whole Namespaces to include Phase Status.
```

Comment:
Does the last really sentence make sense here now? Delete?

Reply:
I think it is still correct, we watch all Namespaces and we watch the full objects (not just the metadata).

Reply:
It is still correct, but not really relevant here in the code. And you removed a corresponding comment for configmaps "Only cache metadata". But just a nit, so feel free to ignore.

Reply:
Indeed, most of this namespacing logic has been moved to `cmd/trust-manager/app/app.go`.
I think the comment is probably still fine.
Also, I did change the ConfigMap and Secret watches so they watch full objects instead of metadata only in the "trusted" namespace.